modules/hcp-support-matrix.adoc

```diff
@@ -17,6 +17,8 @@ Any supported standalone {product-title} cluster can be a management cluster.
 [NOTE]
 ====
 A single-node {product-title} cluster is not supported as a management cluster. If you have resource constraints, you can share infrastructure between a standalone {product-title} control plane and {hcp}. For more information, see "Shared infrastructure between hosted and standalone control planes".
+
+In addition, a hosted cluster supports only one infrastructure platform. For example, you cannot create multiple node pools on different infrastructure platforms.
```

Comment:
I think this should be added under "Hosted cluster platform support" https://docs.openshift.com/container-platform/4.18/hosted_control_planes/hcp-prepare/hcp-requirements.html#hcp-matrix-platform_hcp-requirements